### PR TITLE
OMEGA-168: Adjust metering point page to new design

### PR DESCRIPTION
**Pull request by eTallang**
OMEGA-168

## What is the reason for this change?

Previously, the customer name and navigational tabs were visible on the metering point page, which is confusing.
image

### What is the expected result?

This PR changes this by hiding the title, customer no, and tabs when on the metering point page. It also adds the selected metering point to the breadcrumbs for navigational purposes.

[Url or screenshot if possible]
image

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |        |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?

- [ ] Check that the implementation complies with the design in [Figma](https://www.figma.com/file/sJHoScmQRq0hlr6AcHI9WB/Elvia.no?node-id=11089%3A77531).
- [ ] Check that the breadcrumbs update when selecting other metering points, and when navigating back and forth.
- [ ] Check that the breadcrumbs can be used for navigation.
- [ ] Check that the page is responsive

### CODE REVIEW

**Review by lobunto — DISMISSED** [review]

**Comment by t0ff3r** [comment]
If you only have 1 meterpoint you should get it in the tab for "Min måler" and not get the table. Right?

File: packages/web/src/templates/CustomerDetail/CustomerDetail.tsx [hunk 1]
```diff
@@ -50,6 +57,14 @@ export const CustomerDetail: React.FC<CustomerDetail> = ({
...
   ]
 
+  const getContractAddressWithId = (id: string) => {
+    const contract = contracts.find(c => c.id === id)
+    const address = contract?.meteringPoint.address
+    return `${address?.streetName} ${
+      address?.buildingNumber
+    }${address?.letter || ''}`
+  }
+
```
Comment (marasva) [inline packages/web/src/templates/CustomerDetail/CustomerDetail.tsx:67]: Should / could this be written in extractDataHelper.tsx or something?

Comment (lobunto) [reply packages/web/src/templates/CustomerDetail/CustomerDetail.tsx:67]: There is actually a Ramda function in extractDataHelper.tsx that does exactly this: `getCustomerAdress()`

**Comment by eTallang** [comment]
> If you only have 1 meterpoint you should get it in the tab for "Min måler" and not get the table. Right?

True, but this will be solved through OMEGA-147

**Review by t0ff3r — APPROVED** [review]

**Pull request merged**